server/.env

```diff
@@ -0,0 +1 @@
+DB_URL = postgres://elena:EryWSIiXXDFufKrilWGtm1BOeO2tOrFH@dpg-ck27hvn03lhc7389didg-a.frankfurt-postgres.render.com/dbvideoselena
```

Comment:
You probably didn't mean to check this file in - I'd remove it, add it to your `.gitignore`, and change your password :)

Reply:
Done. Thank you)